pkg/releasesync/releasesync.go

```diff
@@ -169,6 +204,24 @@ func releaseDetailsToDB(architecture string, tag ReleaseTag, details ReleaseDeta
 	release.PullRequests = changelog.PullRequests()
 	release.JobRuns = releaseJobRunsToDB(details)
 
+	// set forced flag
+	failedBlocking := false
+
+	for _, jRun := range release.JobRuns {
+		if jRun.State == failed {
+			if jRun.Kind == "Blocking" {
+				failedBlocking = true
+				break
+			}
+		}
+	}
+
+	if release.Phase == "Accepted" {
+		release.Forced = failedBlocking
+	} else if release.Phase == "Rejected" {
+		release.Forced = !failedBlocking
```

Comment:
Oof this is a tough one. Technically we could have force rejected before any job actually completed, and virtually all force rejections would be while jobs are still running. This is arguably best effort and probably good enough, force accepted is all we really care about I think. We could change it to just ForceAccepted, but this is also pretty good as is I think.

Or should we try to detect all blocking jobs not completed and phase Rejected on the payload?

Reply:
Thinking about this some, if our phase is 'Rejected'  and failedBlocking === false (we didn't have any blocking jobRuns with a state of 'Failed') we would set the release.Forced flag to true which is what I think we want in this case.  If the release phase is 'Rejected' we consider it forced if we have no failed blocking jobs.

Reply:
Ok that would lead to one more possible issue, would we get Forced=true if *some* blocking jobs failed, but others did not complete? this might not even be worth worrying about, it doesn't matter if it was force rejected at that point really, a blocker filed, it was going to get rejected anyhow. I think this might be good to go.